src/main/java/omero/gateway/Gateway.java

```diff
@@ -1099,17 +1099,25 @@ private SessionWrapper createSession(LoginCredentials c)
             }
         }
         if (!connected) {
-            try {
-                if (args != null) {
+            if (args != null) {
+                try {
                     entryEncrypted = secureClient.createSession();
-                } else {
+                } catch(Exception e) {
+                    log.warn(this, new LogMessage(
+                            "Could not login with command line arguments.", e));
+                    entryEncrypted = secureClient.createSession(c.getUser()
+                            .getUsername(), c.getUser().getPassword());
```

Comment:
Argh, I forgot to remove this line again, it's not necessary. Will do now, but can still be tested now.